.github/CONTRIBUTING.md

```diff
@@ -540,7 +540,113 @@ For more on good git commits, see (among _many_ other):
+
+[keepachangelog.com](https://keepachangelog.com) has a neat list of good verbs to start out with, you can use as inspiration for your title:
+
+* `Added` for new features
+* `Changed` for changes in existing functionality 
+* `Deprecated` for soon-to-be removed features 
+* `Removed` for now removed features 
+* `Fixed` for any bug fixes 
+
+Example titles that are worthy of the changelog: 
+
+* "Added default styling for text links"
+* "Removed kirby-toolbar"
+* "Fixed kirby-radio-group TypeError"
+
+### The good pull-request uses the template
+
+As with writing [good issues](#issue), do not ignore the template text when creating a pull-request. 
+
+It provides a checklist to help you follow [the process of contributing](#the-process-of-contributing), ease collaboration by providing structure and reminds you to provide information the reviewer might be looking for. 
+
+Filling out the following part of the template properly, will cause the related issue to be closed automatically once the pull-request has been merged: 
+
+> This PR closes # (reference issue number here)
+
+For example: 
+
+> This PR closes [#1427](https://github.com/kirbydesign/designsystem/issues/1427)
+
+If the pull-request solves more than one issue also mention these such that they also are closed automatically: 
+
+> This PR closes [#1427](https://github.com/kirbydesign/designsystem/issues/1427), closes [#1428](https://github.com/kirbydesign/designsystem/issues/1428), closes [#1429](https://github.com/kirbydesign/designsystem/issues/1429)
+
+### The good pull-request has a logical commit history
+
+Maintaining a logical commit history constructed using [good commits](#commit) helps the reviewer easily get an overview and understanding of the changes contained in a pull-request.
+
+If you are having trouble maintaining a clean commit history, you could eventually try writing [pre-emptive commit comments](https://arialdomartini.wordpress.com/2012/09/03/pre-emptive-commit-comments/). This consists of writing down the commit comment, before writing your code. This can help you stay focused on a single task at a time. 
+
+![](images/good-commit-history.png)
+
+### The good pull-request only alters files related to the issue at hand
+
+While you might get the urge to be a good girl/boy scout and reorder imports or fix formatting in files not related to your issue, don't.
+
+While it is very much appreciated that you want to leave the code better than you found it, doing so in unrelated files will cause the git history of the project to be illogical. 
+For example, it would not make sense if the imports for the list component located in `list.component.ts` is reordered as part of a pull-request concerning optimizing a build step in the CI/CD pipeline. 
+
+Should you stumble upon possible optimizations or things that could be done better, go ahead and create new issues for these.
```

Comment:
Maybe replace

> things that could be done better

with

> things that could be improved

And maybe specify "new issue" as "new housekeeping issue"

Reply:
Reworded to "be improved". I will abstain from using "new housekeeping issue", as it might potentially also be an enhancement, or something third, depending on the type of improvement 😄